Core/src/org/sleuthkit/autopsy/coreutils/AppSQLiteDB.java

```diff
@@ -390,14 +390,17 @@ public boolean tableExists(String tableName) throws TskCoreException {
     private static void findAndCopySQLiteMetaFile(AbstractFile sqliteFile,
             String metaFileName) throws NoCurrentCaseException, TskCoreException, IOException {
 
+        // Do not look for metaFile if this is a carved directory
+        if(sqliteFile.getParentPath().contains("carvedfiles")) {
```

Comment:
This can be an exact match. I think .equals("$carvedfiles") - but we should test.